crates/control/src/behavior/support.rs

```diff
@@ -56,10 +56,14 @@ fn support_pose(
         Side::Right => FRAC_PI_4,
     }) * -(Vector2::x() * distance_to_ball);
     let supporting_position = ball.ball_in_field + offset_vector;
-    let clamped_x = match world_state.filtered_game_state {
+    let clamped_x = match world_state
+        .filtered_game_controller_state
+        .map(|filtered_game_controller_state| filtered_game_controller_state.game_state)
```

Comment:
as this is used so often, you may introduce a function for that?

Reply:
We will fix the bug in the framework. See Issue #668

Reply:
Extracted into variable as discussed